client/my-sites/email/add-mailboxes/get-event-name.ts

```diff
@@ -0,0 +1,22 @@
+import { EmailProvider } from 'calypso/my-sites/email/form/mailboxes/types';
+
+export const EVENT_CONTINUE_BUTTON_CLICK = 'continue_button_click';
+export const EVENT_CANCEL_BUTTON_CLICK = 'cancel_button_click';
+
+export type EventName = typeof EVENT_CONTINUE_BUTTON_CLICK | typeof EVENT_CANCEL_BUTTON_CLICK;
+
+export const getEventName = ( provider: EmailProvider, genericName: EventName ): string => {
```

Comment:
I find it a bit confusing to have `genericName` defined as an `EventName`: I would have expected this parameter to be named `eventName` to match the name of the type. I think we could address this by renaming this function, and making it clear that it returns the name of a Tracks event:

```suggestion
export const getTracksEventName = ( provider: EmailProvider, eventName: EventName ): string => {
```